src/bindgen/cdecl.rs

```diff
@@ -14,7 +14,7 @@ use crate::bindgen::{Config, Language};
 // http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf
 
 enum CDeclarator {
-    Ptr(bool),
```

Comment:
Can we name these? One unnamed bool is bad enough, but two is super easy to mess up, so:

```
Ptr { is_const: bool, non_null: bool }
```

Though it seems like it may be better to keep `nullable: bool` rather than `non_null`.

Reply:
Done